### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by hakujitsu**
**Changes made**
- Add a labelled diagram of our UI
- Add detailed examples for the CRUD Sale commands
- Add a short explanation of the `sale breakdown` command

**Future changes**
- Diagrams for the various examples have not been added, but will be at a later date
- An example will be written for the `sale breakdown` command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -306,22 +277,117 @@ Format: `sale list [c/CONTACT_INDEX] [m/MONTH y/YEAR]`
...
+    
+2. The **Result Box** will display a message of the number of contacts listed and the **Contact List** updates to
+ show only contacts with the name 'Bernice'. Identify the correct index that corresponds to Bernice Yu, which is 1 in this example.
+ 
+3. Type `sale list c/1` in the **Command Box** and press <kbd>Enter</kbd>.
+
+4. The **Result Box** will display a message noting that the command was successful, and the **Sale List** will
+ show a list of sales made to Bernice Yu. Identify the index of the sale to be deleted. Let us assume it is at the third index.
+
+5. Type `sale delete s/4` in the command box, and press Enter to execute it.
```
Comment (sebastiantoh) [inline docs/UserGuide.md]:
```suggestion
5. Type `sale delete s/3` in the command box, and press <kbd>Enter</kbd> to execute it.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -306,22 +277,117 @@ Format: `sale list [c/CONTACT_INDEX] [m/MONTH y/YEAR]`
...
+ 
+    <img src="images/meeting-delete/meeting-delete.png" alt="Enter command 'meeting delete'" width="900px">
```
Comment (sebastiantoh) [inline docs/UserGuide.md]:
```suggestion
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -306,22 +277,117 @@ Format: `sale list [c/CONTACT_INDEX] [m/MONTH y/YEAR]`
...
+**Examples:**
+Suppose you have just received the good news that Bernice Yu would like to buy 20 Guitars instead of the 10 she previously specified. 
+Here's how you can edit her sale in StonksBook:
+ 
+1. Identify the index corresponding to the sale to be edited in the sale list. If you have a large list of
+  sales, it may be convenient to filter for all sales made to Bernice Yu. Assuming that Bernice Yu is the 2nd contact
+   currently displayed in the contact list, you can type `sale list c/2` and the sale list will update to show
+    only sales made to Bernice Yu.
+    
+2. Suppose the sale to be edited is at the third index in the meeting list. Then, type `sale edit s/3 q/20` in the command box, and press <kbd>Enter</kbd> to execute it.
```
Comment (sebastiantoh) [inline docs/UserGuide.md]:
```suggestion
2. Suppose the sale to be edited is at the third index in the sale list. Then, type `sale edit s/3 q/20` in the command box, and press <kbd>Enter</kbd> to execute it.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -306,22 +277,117 @@ Format: `sale list [c/CONTACT_INDEX] [m/MONTH y/YEAR]`
...
+  
+3. You should see that the meeting has been updated to reflect this new scheduled date.
```
Comment (sebastiantoh) [inline docs/UserGuide.md]:
```suggestion
3. You should see that the sale has been updated to reflect this new quantity.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -306,22 +277,117 @@ Format: `sale list [c/CONTACT_INDEX] [m/MONTH y/YEAR]`
...
-* `sale list m/6 y/2020` lists all sales whose associated date is within June 2020.
+**Example:** Let's say you want to view all sales made to Bernice Yu. This is what you need to do:
+
+1. Type `contact find bernice` in the **Command Box** and press <kbd>Enter</kbd>. 
+    - This is to identify the contact index corresponding to Bernice Yu.     
+    
+2. The **Result Box** will display a message of the number of contacts listed and the **Contact List** updates to
+ show only contacts with the name 'Bernice'. Identify the correct index that corresponds to Bernice Yu, which is 1 in this example. 
+ 
+3. Type `sale list c/1` in the **Command Box** and press <kbd>Enter</kbd>.\
```
Comment (sebastiantoh) [inline docs/UserGuide.md]: Does this backslash affect anything?

**Review by sebastiantoh — COMMENTED** [review]
Thanks for doing up the labelled UI diagram!

File: docs/UserGuide.md (hunk 5 above)
Comment (hakujitsu) [reply docs/UserGuide.md]: I've deleted the typo!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -310,22 +281,116 @@ Shows a list of sales.
...
+    
+2. The **Result Box** will display a message of the number of contacts listed and the **Contact List** updates to
+ show only contacts with the name 'Bernice'. Identify the correct index that corresponds to Bernice Yu, which is 1 in this example.
+ 
+3. Type `sale list c/1` in the **Command Box** and press <kbd>Enter</kbd>.
+
+4. The **Result Box** will display a message noting that the command was successful, and the **Sale List** will
+ show a list of sales made to Bernice Yu. Identify the index of the sale to be deleted. Let us assume it is at the third index.
+
+5. Type `sale delete s/3` in the command box, and press <kbd>Enter</kbd> to execute it.
```
Comment (sebastiantoh) [inline docs/UserGuide.md]:
```suggestion
5. Type `sale delete s/3` in the **Command Box**, and press <kbd>Enter</kbd> to execute it.
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -310,22 +281,116 @@ Shows a list of sales.
...
+**Examples:**
+Suppose you have just received the good news that Bernice Yu would like to buy 20 Guitars instead of the 10 she previously specified. 
+Here's how you can edit her sale in StonksBook:
+ 
+1. Identify the index corresponding to the sale to be edited in the sale list. If you have a large list of
+  sales, it may be convenient to filter for all sales made to Bernice Yu. Assuming that Bernice Yu is the 2nd contact
+   currently displayed in the contact list, you can type `sale list c/2` and the sale list will update to show
+    only sales made to Bernice Yu.
+    
+2. Suppose the sale to be edited is at the third index in the sale list. Then, type `sale edit s/3 q/20` in the command box, and press <kbd>Enter</kbd> to execute it.
```
Comment (sebastiantoh) [inline docs/UserGuide.md]:
```suggestion
2. Suppose the sale to be edited is at the third index in the sale list. Then, type `sale edit s/3 q/20` in the **Command Box**, and press <kbd>Enter</kbd> to execute it.
```

**Review by sebastiantoh — APPROVED** [review]
sorry just spotted some more minor formatting issues

**Pull request merged**